test/helpers/policygen/models.go

```diff
@@ -679,3 +680,45 @@ func (t *TestSpec) ExecTest() error {
 	}
 	return nil
 }
+
+// TestSpecsGroup is a group of different TestSpec
+type TestSpecsGroup []*TestSpec
+
+// CreateAndApplyManifests creates all of the pods manifests and applies those
+// manifest to the given kubernetes instance.
+func (tg TestSpecsGroup) CreateAndApplyManifests(kub *helpers.Kubectl) {
+	completeManifest := "/tmp/data.yaml"
```

Comment:
We should use a temporary file https://golang.org/pkg/io/ioutil/#TempFile

Reply:
Hi @aanm the issue with the `TempFile` is that it creates the file. In this case the file will be into the VM and we can't use `TempFile` due the execution happens on the Jenkins box. 

Regards